src/psyclone/dynamo0p3_builtins.py

```diff
@@ -130,6 +130,8 @@ def __init__(self):
             create_name(root_name="df",
                         context="PSyVars",
                         label="dof_loop_idx")
+        # Builtins do not accept quadrature
+        self.qr_rules = {}
```

Comment:
The `return False` in `def qr_required(self):` appears not to be tested. Could you please check this?

Reply:
Yes, I agree. I've removed the method.

Reply:
...and that broke the examples. I've now added a test that covers this method (use of OpenACC with a built-in).